packages/database/src/database/tokenBoundAccounts.ts

```
@@ -0,0 +1,49 @@
+    if (size > 200) { size = 200 }
+    let start = page * size
+    filter.skip = start
+    filter.take = size
+    //filter.orderBy = {}
+  }
+  let data = await prismaClient.tokenBoundAccount.findMany(filter)
+  return data
+}
+
+export async function newAccount(data:Prisma.TokenBoundAccountCreateInput): Promise<TokenBoundAccount> {
+  let result = await prismaClient.tokenBoundAccount.create({data})
```

Biome

[error] This let declares a variable that is only assigned once.

'result' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)